pkg/zeek/escape.go

```diff
@@ -23,6 +25,30 @@ func Escape(data []byte) string {
 	return string(buf)
 }
 
+// EscapeUtf8 does the very same non-standard formatting of mixed-binary strings
+// that zeek does.  There is no way to disambiguate between random binary data
```

Comment:
If there is a pointer to zeek's description/documentation of how they format that would be good to have here. Or just a link to the relevant code...

Reply:
Just poked around.  It's documented here... https://docs.zeek.org/en/stable/scripts/base/frameworks/logging/writers/ascii.zeek.html
how to allow UTF8 code points, i.e., non-ASCII, to pass through in the ASCII log writer :-)
I'll update this comment.